lib/web/fetch/headers.js

```diff
@@ -576,8 +577,21 @@ class Headers {
 
     return this[kHeadersList]
   }
+
+  [util.inspect.custom] (depth, options) {
+    const headerString = Array.from(this[kHeadersList])
+      .map(([name, value]) =>
+          `${name.replace(/\b\w/g, (char) =>
+            char.toUpperCase())}: ${value}`)
+      .join('\n')
+    return 'Headers:\n' + headerString
```

Comment:
```suggestion
    const inspected = util.inspect(this[kHeadersList].entries)
    
    return `Headers ${inspected}`
```

I could even see inspecting the headersList's headers map, slicing the "Map(...) " from it and replacing it with "Headers" which might look better (and be a more accurate representation of how Headers works).

Reply:
thank you I applied this